tests/invalid/Reference_Invalid_2.sysout

```diff
@@ -0,0 +1,3 @@
+../../tests/invalid/Reference_Invalid_2.whiley:0: invalid array expression
+
+^
```

Comment:
Probably related to #590, there we miss the information for wrong-typed `new` expressions.